module/database/UserDatabase.py

```diff
@@ -63,13 +65,15 @@ def changePassword(db, user, oldpw, newpw):
 
         salt = r[2][:5]
         pw = r[2][5:]
-        h = sha1(salt + oldpw)
+        h = sha1((salt + oldpw).encode('utf-8'))
```

Comment:
Codacy found an issue: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5985155)